tt_eager/tt_dnn/op_library/bmm/multi_core_reuse_mcast_1d_optimized/bmm_op_multi_core_reuse_mcast_1d_optimized.cpp

```diff
@@ -139,19 +139,26 @@ operation::ProgramWithCallbacks create_program_mcast_in0(
 
     std::vector<uint32_t> in0_sender_compile_time_args;
     if (in0_is_sharded) {
+        uint32_t in0_shard_width_in_tiles = in0_buffer->shard_spec().shape()[1] / 32;
+        uint32_t in0_shard_height_in_tiles = in0_buffer->shard_spec().shape()[0] / 32;
```

Comment:
Use TILE_HEIGHT and TILE_WIDTH and avoid hard-coded 32.

Reply:
Also probably better to move it to where the other constants are for cleanliness.